make/_shared/cert-manager/01_mod.mk

```
@@ -0,0 +1,17 @@
+# Copyright 2023 The cert-manager Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cert_manager_crds := $(bin_dir)/scratch/cert-manager-crds.yaml
+$(cert_manager_crds): | $(bin_dir)/scratch
+	curl -sSLo $@ https://github.com/cert-manager/cert-manager/releases/download/$(cert_manager_version)/cert-manager.crds.yaml
```

suggestion (not a blocker for this PR but maybe something wider): If we don't include a version number in the file we create, it won't be recreated when we bump `cert_manager_version`.

Could we do something like this?

```suggestion
cert_manager_crds := $(bin_dir)/scratch/cert-manager-crds-$(cert_manager_version).yaml
$(cert_manager_crds): | $(bin_dir)/scratch
	curl -sSLo $@ https://github.com/cert-manager/cert-manager/releases/download/$(cert_manager_version)/cert-manager.crds.yaml
```

Of course this means that the location is no longer constant but we can always pass the value of `cert_manager_crds` as a variable, or create a new `$(bin_dir)/scratch/cert-manager-crds.yaml` target which depends on `$(bin_dir)/scratch/cert-manager-crds-$(cert_manager_version).yaml`

---

good catch, I'll change this. This is part of https://github.com/cert-manager/makefile-modules, so the fix will also improve approver-policy.

---

@SgtCoDFish https://github.com/cert-manager/makefile-modules/pull/17

---

We can merge that PR after this PR (or before).